modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -81,74 +89,77 @@
 			<#list entries as entry>
 				<#if entry?has_content>
 					<#assign
-						portalURL=portalUtil.getLayoutURL(themeDisplay)
-						productId=entry.getClassPK() + 1
-						product=restClient.get("/headless-commerce-admin-catalog/v1.0/products/" + productId + "?nestedFields=productSpecifications,attachments" )
-						productAttachments=product.attachments![]
+						portalURL = portalUtil.getLayoutURL(themeDisplay)
+						productId = entry.getClassPK() + 1
+						product = restClient.get("/headless-commerce-admin-catalog/v1.0/products/" + productId + "?nestedFields=productSpecifications,attachments")
+						productAttachments = product.attachments![]
 						productCategories=product.categories![]
-						productDescription=stringUtil.shorten(htmlUtil.stripHtml(product.description.en_US!""), 150, "..." )
-						productSpecifications=product.productSpecifications![]
-						productURL=portalURL?replace("solutions-marketplace", "p" ) + "/" + product.urls.en_US />
+						productDescription = stringUtil.shorten(htmlUtil.stripHtml(product.description.en_US!""), 150, "...")
+						productSpecifications = product.productSpecifications![]
+						productURL = portalURL?replace("solutions-marketplace", "p") + "/" + product.urls.en_US
+					/>
 
 					<a class="solution-search-results-card bg-white border-radius-medium d-flex flex-column mb-0 rounded text-dark text-decoration-none" href=${productURL}>
-						<div class="align-items-center d-flex image-container justify-content-center">
+						<div class="align-items-center d-flex image-container justify-content-center mb-3">
 							<img
+								width="329" height="180"
```

Comment:
```suggestion
								
```
Since you're declaring a class here, couldn't you use css instead of setting it in the tag?

Reply:
Adjusted !